tfjs-core/src/io/types.ts

```diff
@@ -532,6 +539,12 @@ export interface LoadOptions {
    * With this func you can convert the weight file name to any URL.
    */
   weightUrlConverter?: (weightFileName: string) => Promise<string>;
+
+  /**
+   * Whether to stream the model directly to the backend or cache all its
+   * weights on CPU. Useful for large models.
+   */
+  streamWeights?: boolean;
```

Comment:
this flag is the same name as the function `streamWeights?: () => ReadableStream<ArrayBuffer>`?

Reply:
Yes, but they're part of different interfaces. One is on the `ModelArtifacts` and is the actual function that, when called, will start streaming the weights. It's internal, and users shouldn't really need to access it.

The other is on the `LoadOptions` interface, and it's used to configure whether the model loader should stream the weights or load them normally. It's exposed to the user.

I can rename the one on `ModelArtifacts` to something like `startWeightsStream` or `streamWeightsData`.